verilog/analysis/verilog_linter_configuration.cc

```diff
@@ -262,6 +268,15 @@ std::set<analysis::LintRuleId> LinterConfiguration::ActiveRuleIds() const {
   return result;
 }
 
+void LinterConfiguration::GetRuleBundle(RuleBundle *rule_bundle) const {
+  // Clear the vector to overwrite any existing value.
+  rule_bundle->rules.clear();
+
+  for (const auto &rule_pair : configuration_) {
+    rule_bundle->rules[rule_pair.first] = rule_pair.second;
```

Comment:
couldn't this whole function just be a `rule_bundle_->rules = configuration_` ?

Reply:
My C++ is rusty (thanks for your patience). Yes it would be.